packages/nodes-base/nodes/MQTT/MqttTrigger.node.ts

```diff
@@ -51,12 +38,13 @@ export class MqttTrigger implements INodeType {
 		],
 		properties: [
 			{
-				displayName: 'Topics',
-				name: 'topics',
+				displayName: 'Topic',
+				name: 'topic',
```

Comment:
This will be a breaking change and would require a new node version if you wanted to do this